### ISSUE TITLE
Bungeni Search Indexer and Site Search

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#632]
```
The site search functionality in bungeni is missing.

TO DO :
 -- Add a site-search functionality with a search box on the top right hand corner of every page in the site
 -- Add an advanced search option to allow searching within specific content types - and with date filters
```

Original issue reported on code.google.com by `ashok.ha...@gmail.com` on 17 Jun 2010 at 7:36
